### ISSUE TITLE
Loading state didn't changed correctlly when using custom render props

### ISSUE THREAD

**Issue by abramstyle** [smooth-code/loadable-components#63]
Hi, thanks for this amazing package. I have a problem when I using custom render function.

If using render props, the loading didn't change even if component is already loaded.

Here are my code:

```javascript
const LoadablePosts = loadable(() => import(/* webpackChunkName: "posts" */'./Posts'), {
  render(renderProps) {
    const { Component, loading, ownProps } = renderProps;
    const { store } = ownProps;
    // 'loading' is always be 'true'
    if (loading) {
      return <div className="loading">Loading Desktop Posts...</div>;
    }

    store.replaceReducer(nextReducer());
    return (<Component {...ownProps} />);
  },
});

```

**Comment by neoziro** [comment]
You are right, I think there is a bug.

https://github.com/smooth-code/loadable-components/blob/master/src/loadable.js#L52

`loading` should be determined from `LoadableComponent.Component !== null`

PR welcome.

**Comment by abramstyle** [comment]
Here is the PR https://github.com/smooth-code/loadable-components/pull/64, tests are all passed.

**Comment by neoziro** [comment]
Fixed in #64, thanks @abramstyle
